manifests/base/tasks/onboarding.yaml

```diff
@@ -0,0 +1,50 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: task
+metadata:
+  name: run
+spec:
+  params:
+    - name: PAYLOAD
+      type: string
+      default: ""
+  steps:
+    - name: clone-repo
+      image: opf-toolbox
+      script:
+        #!/usr/bin/bash
+        
+        echo "Cloning repo"
+        git clone https://github.com/tssala23/ReadIssuesTest.git
+        cd ReadIssuesTest
+        
+    - name: create-patch
+      image: opf-toolbox
+      script:
+        #!/usr/bin/bash
+        
+        echo '$PAREMETER_FOR_THE_JSON_PAYLOAD' | yq e "" -P - > /tmp/data.yaml
+        mustache /tmp/data.yaml /mnt/config/robozome/onboarding.patch > /tmp/onboarding.patch
+        
+    - name: apply-push-patch
+      image: opf-toolbox
+      script:
+        #!/usr/bin/bash
+        
+        NAMESPACE=$(yq e ".namespace" /tmp/data.yaml)
+        CLUSTER=$(yq e ".cluster" /tmp/data.yaml)
+        GROUP=$(yq e ".group" /tmp/data.yaml)
+        DESCRIPTION=$(yq e ".description" /tmp/data.yaml)
+        USERS=$(yq e ".users" /tmp/data.yaml)
+        QUOTA=$(yq e ".quota" /tmp/data.yaml)
+        GPGKey=$(yq e ".gpg" /tmp/data.yaml)
```

Comment:
What is the purpose of these variables here?

Reply:
All removed, was storing to use later but not just accessing data when needed